dist-build/android.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/sh
+
+# uncomment below to disable the minimal build
```

Comment:
Distributed files are not supposed to be modified. Maybe allow these parameters to be set on the command-line instead?

Reply:
I'm happy to make that change when I have time. The old scripts didn't have any easy way to configure them and this just seemed like the fastest way.

Reply:
Can you clarify how you'd prefer this? command line argument? prompt after running the script? 

If I use command line arguments and there are defaults many people will not realise what defaults are being set which might bite them if they use functionality that isn't included in a minimal build (though that default could be changed). With no defaults the scripts of course won't do anything unless arguments are specified. What are your thoughts on using command line arguments if specified and if none are specified printing out what the defaults are and giving a Y/N prompt? Without a prompt any messaging will fly by as the configure scripts get run and the compilations happen. The other option is prompting for different options on stdin after the program is run or potentially only doing this if options are missing though this can be painful for anyone running the script from a script.

I guess my gut is how will anyone understand the commandline arguments without looking at the script or not having defaults and having the script complain if nothing is provided or having a system of prompts to confirm anything that isn't provided. If someone does look at the script isn't setting the variables in the script easier for everyone? 

So how about I go with script variables for now and take on a future task to make a more intelligent command line argument system that uses flags and has prompts if anything is missing?